config/environments/staging.rb

```diff
@@ -53,7 +53,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  # config.force_ssl = true / Disabled since Cloud66 manages SSL and DNS redirects for us.
```

Comment:
Is there an environment variable we can use to determine if we're on Heroku or Cloud66? Then we have a way to deploy the code on both places and have a smoother cutover by essentially feature flagging the code changes.